node/src/components/network.rs

```diff
@@ -203,26 +212,23 @@ where
     state: ComponentState,
 }
 
-#[derive(DataSize)]
 struct ChannelManagement {
     /// Channel signaling a shutdown of the network.
     // Note: This channel is closed when `Network` is dropped, signalling the receivers that
     // they should cease operation.
-    #[data_size(skip)]
+    #[cfg(test)]
     shutdown_sender: Option<watch::Sender<()>>,
     /// Join handle for the server thread.
-    #[data_size(skip)]
+    #[cfg(test)]
     server_join_handle: Option<JoinHandle<()>>,
 
     /// Channel signaling a shutdown of the incoming connections.
     // Note: This channel is closed when we finished syncing, so the `Network` can close all
     // connections. When they are re-established, the proper value of the now updated `is_syncing`
     // flag will be exchanged on handshake.
-    #[data_size(skip)]
-    close_incoming_sender: Option<watch::Sender<()>>,
+    _close_incoming_sender: Option<watch::Sender<()>>,
```

Comment:
why retain the field if it is now unused?

Reply:
It is used for testing.